### PR TITLE
ART-10885 Populate end_time for failed builds with task completion time

### PR DESCRIPTION
**Pull request by locriandev**

### CODE REVIEW

File: doozer/doozerlib/distgit.py [hunk 1]
```diff
@@ -1351,7 +1356,7 @@ def update_konflux_db(self, build_info, outcome, build_pipeline_url='', scratch=
                 self.logger.info('Storing failed Brew build info for %s in Konflux DB', self.metadata.name)
                 build_record_params.update({
                     'start_time': datetime.now(tz=UTC),  # TODO: store start time from taskID
```
Comment (thegreyd) [inline doozer/doozerlib/distgit.py:1358]: change this as well?

Comment (locriandev) [reply doozer/doozerlib/distgit.py]: done

**Comment by locriandev** [comment]
We're not storing failed builds for RPMs. olm-bundle needs this too, let me do it

/hold

**Comment by locriandev** [comment]
@thegreyd adjusted failed olm bundle builds too, see 09f6dba5c4bde68a8afa52b0cb1978209103abb0

**Review by joepvd — APPROVED** [review]
/lgtm

**Pull request merged**